addons/pkg-maven/common/src/main/java/org/commonjava/indy/pkg/maven/change/sl/SnapshotFilter.java

```diff
@@ -13,13 +13,13 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.commonjava.indy.core.change.sl;
+package org.commonjava.indy.pkg.maven.change.sl;
```

Comment:
I was looking at this one before, and I think it's unused. I think we can remove it.